src/Ocelot/Requester/MessageInvokerPool.cs

```diff
@@ -31,17 +38,15 @@ public HttpMessageInvoker Get(DownstreamRoute downstreamRoute)
     }
 
     public void Clear() => _handlersPool.Clear();
-
-    /// <summary>
-    /// TODO This should be configurable and available as global config parameter in ocelot.json.
-    /// </summary>
-    public const int DefaultRequestTimeoutSeconds = 90;
+    
+    private readonly int _defaultRequestTimeoutSeconds;
+    
     private int _requestTimeoutSeconds;
 
     public int RequestTimeoutSeconds
     {
-        get => _requestTimeoutSeconds > 0 ? _requestTimeoutSeconds : DefaultRequestTimeoutSeconds;
-        set => _requestTimeoutSeconds = value > 0 ? value : DefaultRequestTimeoutSeconds;
+        get => _requestTimeoutSeconds > 0 ? _requestTimeoutSeconds : _defaultRequestTimeoutSeconds;
+        set => _requestTimeoutSeconds = value > 0 ? value : _defaultRequestTimeoutSeconds;
```

Comment:
These changes are not required if `DefaultRequestTimeoutSeconds` will be converted to a property.